BlogTemplate.Tests/Pages/NewTests.cs

```diff
@@ -1,29 +1,33 @@
 using BlogTemplate._1.Models;
+using BlogTemplate._1.Pages;
 using BlogTemplate._1.Tests.Fakes;
+using BlogTemplate._1.Services;
 using Xunit;
+using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace BlogTemplate._1.Tests.Pages
 {
-    class NewTests
+    public class NewTests
     {
         [Fact]
         public void SavePost_NoExcerptIsEntered_AutoGenerateExcerpt()
         {
             IFileSystem fakeFileSystem = new FakeFileSystem();
             BlogDataStore testDataStore = new BlogDataStore(fakeFileSystem);
+            ExcerptGenerator testExcerptGenerator = new ExcerptGenerator();
+            testExcerptGenerator.maxLength = 5;
+            SlugGenerator testSlugGenerator = new SlugGenerator(testDataStore);
 
-            Post newPost = new Post
-            {
-                Title = "Title",
-                Body = "This is the body of my post",
-            };
+            NewModel model = new NewModel(testDataStore, testSlugGenerator, testExcerptGenerator);
+            model.PageContext = new PageContext();
+            model.OnGet();

```

Comment:
No need to call `OnGet()` here

Reply:
The test fails when I remove the model.OnGet() line.

Reply:
I get the message: "System.NullReferenceException : Object reference not set to an instance of an object," referring to NewPost in model.NewPost.Title. This is because the OnGet method in the New page class creates an instance of a NewPostViewModel, called NewPost.
Does that mean I should keep the model.OnGet() line in the test, write something else in the test to fix the problem when I remove model.OnGet(), or change something about the New page class itself?